backend/api/Services/ActivityService.cs

```diff
@@ -0,0 +1,122 @@
+using System.Collections.Generic;
+using System.Security.Claims;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Pims.Dal.Entities;
+using Pims.Dal.Helpers.Extensions;
+using Pims.Dal.Repositories;
+using Pims.Dal.Security;
+
+namespace Pims.Api.Services
+{
+    public class ActivityService : BaseService, IActivityService
+    {
+        private readonly ILogger _logger;
+        private readonly IActivityRepository _activityRepository;
+        private readonly IActivityTemplateRepository _activityTemplateRepository;
+        private readonly IDocumentService _documentService;
+        private readonly INoteService _noteService;
+
+        public ActivityService(
+            ClaimsPrincipal user,
+            ILogger<ActivityService> logger,
+            IActivityRepository activityRepository,
+            IActivityTemplateRepository activityTemplateRepository,
+            INoteService noteService,
+            IDocumentService documentService)
+            : base(user, logger)
+        {
+            _logger = logger;
+            _activityRepository = activityRepository;
+            _activityTemplateRepository = activityTemplateRepository;
+            _noteService = noteService;
+            _documentService = documentService;
+        }
+
+        public PimsActivityInstance GetById(long id)
+        {
+            _logger.LogInformation("Getting activity {id}", id);
+            this.User.ThrowIfNotAuthorized(Permissions.NoteView);
+
+            // _user.ThrowIfNotAuthorized(Permissions.ResearchFileView);
```

Comment:
?

Reply:
This was from when I was using placeholder permissions, will remove